athena/map/Vector.tsx

```diff
@@ -1,4 +1,6 @@
 import sortBy from '@deities/hephaestus/sortBy.tsx';
+import { DrawingMode } from '../../hera/editor/Types.tsx';
```

Comment:
We can't import from `hera` in the `athena` package since `athena` also runs on the server without access to React. I think we need to define a `VectorMirror` (?) type with just `'horizontal' | 'vertical' | 'diagonal'` and then use that as a base for `DrawingMode` in the editor types.

Reply:
This should be solved, since I moved `vector.mirror` inside `hera`: https://github.com/nkzw-tech/athena-crisis/commit/349ef4cd7428ee57e22736e50dc722a322a07e2a